src/include/daos/tests_lib.h

```diff
@@ -222,4 +224,81 @@ dts_append_config(char buf[DTS_CFG_MAX], const char *format, ...)
 		buf[DTS_CFG_MAX - 1] = 0;
 }
 
+/**
+ * List all pools created in the specified DAOS system.
+ *
+ * \param dmg_config_file
+ *			[IN]	DMG config file
```

Comment:
we should really fix our docs to use \param[in] rather than \param var[in] as the former is parsable by doxygen

Reply:
Probably should just do it all at once, right? I was just copying the existing style.